.circleci/config.yml

```diff
@@ -741,40 +722,6 @@ jobs:
           paths:
             - ~/docker/nucypher.tar
 
-  test_deploy:
-    <<: *python_37_base
-    steps:
-      - checkout
-      - run:
-          name: Install Twine
-          command: pip3 install --user twine
-      - run:
-          name: Verify git tag == __version__ (Test Deploy)
-          command: python setup.py verify
-      - run:
-          name: Initialize .pypirc (Test Deploy)
-          command: |
-            echo -e "[distutils]" >> ~/.pypirc
-            echo -e "index-servers = " >> ~/.pypirc
-            echo -e "    pypi" >> ~/.pypirc
-            echo -e "    testpypi" >> ~/.pypirc
-            echo -e "" >> ~/.pypirc
-            echo -e "[testpypi]" >> ~/.pypirc
-            echo -e "repository: https://test.pypi.org/legacy/" >> ~/.pypirc
-            echo -e "username: $TEST_PYPI_USERNAME" >> ~/.pypirc
-            echo -e "password: $TEST_PYPI_PASSWORD" >> ~/.pypirc
-      - run:
-          name: Build Python Wheel (Test Deploy)
-          command: make dist
-      - run:
-          name: upload to TestPyPI
-          command: python3 -m twine upload --repository testpypi dist/* --verbose
-      - run:
-          name: pip install from TestPyPI
-          command: |
-            pip3 install --user --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple nucypher --verbose
```

Comment:
As it turns out - this has been doing the wrong thing for a long time - installing an unspecified version from test PyPI results in the *first ever upload*.  From a high-level this test is superfluous since we can smoke-test the distribution without uploading to a remote index server.

Reply:
What a coincidence. I've never seen this job fail and I just experienced now doing a new release:
https://app.circleci.com/pipelines/github/nucypher/nucypher/5062/workflows/824f2879-1250-4945-980a-9caea482425d/jobs/91277